data/yam/agama/auto/sap_default.jsonnet

```diff
@@ -1,6 +1,6 @@
 {
   "product": {
-    "id": "SLES_SAP_16.0",
+    "id": "{{PRODUCT}}",
```

Comment:
this file now is not specific for sap, it is for remote worker I think.

Reply:
So we need a separate profile for remote worker? Currently we use the same profile for all arches. And yes, I will update this profile to be a general file not only for sap, to set SCC_REGCODE instead of SCC_REGCODE_SLES4SAP which will be converted to SAP reg code in function expand_profile_vars. So this one and sle_default.jsonnet will be same profile named to default.jsonnet ?